src/load-translations/load-translations.ts

```diff
@@ -77,6 +77,7 @@ export const loadTranslations = async (
     __ni18n_server__: {
       resources: store,
       ns: namespaces,
+      lng: initialLocale,
```

Comment:
It nextjs complains on undefined, so we should return the property here only if it has value

```suggestion
       ...(initialLocale && { lng: initialLocale }),
```

Reply:
Oh right, my least favorite Next error. Good catch, updated.